stan/math/prim/prob/discrete_range_rng.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef STAN_MATH_PRIM_PROB_DISCRETE_RANGE_RNG_HPP
+#define STAN_MATH_PRIM_PROB_DISCRETE_RANGE_RNG_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/max_size.hpp>
+#include <boost/random/uniform_int_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
+
+namespace stan {
+namespace math {
+
+/** \ingroup prob_dists
+ * Return a discrete random variate for the given lower and upper bounds
```

Comment:
I think it'd help to say that it returns integer(s) between the lower and upper bound(s) inclusive and then elaborate on the return type in the longer doc.  As is, it's very unclear from reading this doc what the return type actually is.   I imagine a lot of the other RNG doc probaby has this same problem.